code/modules/projectiles/guns/special/syringe_gun.dm

```diff
@@ -84,15 +84,28 @@
 	var/syringe_count = syringes.len
 	. += "[initial(icon_state)]_[syringe_count ? clamp(syringe_count, 1, initial(max_syringes)) : "empty"]"
 
+/obj/item/gun/syringe/rifle
+	name = "syringe rifle"
+	desc = "A spring loaded rifle designed to fit syringes, used to incapacitate unruly patients from a distance."
+	icon_state = "syringerifle"
+	lefthand_file = 'icons/mob/inhands/weapons/64x_guns_left.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/64x_guns_right.dmi'
+	inhand_icon_state = "syringerifle"
+	inhand_x_dimension = 64
+	inhand_y_dimension = 64
+	worn_icon_state = null
+	w_class = WEIGHT_CLASS_BULKY
+	slot_flags = ITEM_SLOT_BACK
+	force = 6 //A little bit more robust in melee since it's larger
 /obj/item/gun/syringe/rapidsyringe
```

Comment:
please put at least one space between the last type declaration and the next

Reply:
Done 👍